third_party/rust/bitflags/v1/BUILD.gn

```diff
@@ -3,6 +3,34 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at https://mozilla.org/MPL/2.0/.
 
-group("lib") {
-  public_deps = [ "//third_party/rust/bitflags/v1:lib" ]
+# @generated from third_party/rust/chromium_crates_io/BUILD.gn.hbs by
+# tools/crates/gnrt.
+# Do not edit!
+
+import("//build/rust/cargo_crate.gni")
+
+cargo_crate("lib") {
```

Comment:
I don't see bitflags removed from upstream here so why are we adding it?

Reply:
Fixed in 706d836e6862ec220b601cbba8fa348f2c682894. Thanks for noticing that.